.eslintrc

```diff
@@ -224,7 +225,7 @@ rules:
   no-unsafe-negation: [2]
   no-unused-expressions: [1]
   no-unused-labels: [0]
-  no-unused-vars: [1, {args: after-used, vars: local, argsIgnorePattern: ^_}]
+  no-unused-vars: [0]
```

Comment:
Why disabling this?

Reply:
When `npm run lint` is run, a huge list of `no-unsed-vars` shows up... somehow it's ignoring the globals set at the top. I should have investigated more instead of disabling.

Reply:
Oops, I meant the functions are defined but never used... at least not within the same file.

Reply:
running `eslint **/*.js` manually in the master branch produces only one warning for the unused vars.

Reply:
btw I usually simply run `eslint .`

Reply:
<details>
<summary><code>no-unused-vars</code> wall of warnings</summary>

![](https://user-images.githubusercontent.com/136959/43036214-a7f7fe00-8cc2-11e8-9fc9-dfca3829056e.png)
</details>

Reply:
Could be a bug in your version of eslint.
I've been using 4.19.1 for the last few months, and now the latest 5.2.0.